modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -1128,7 +1128,7 @@
 			}
 
 			var eventHandles = [
-				iframeBody.delegate('submit', detachEventHandles, 'form'),
+				iframeBody.delegate('submit', () => {}, 'form'),
```

Comment:
I think we can remove this event delegation, it was called only to detach events.

Reply:
I sent another [PR](https://github.com/liferay-frontend/liferay-portal/pull/611) with this proposal.